playbooks/provision-idm-server/main.yml

```diff
@@ -3,41 +3,29 @@
 - import_playbook: ../prep.yml
   when:
   - rhsm_register|default(False)
-  tags:
-  - 'never'
-  - 'install'
 
 - import_playbook: ../osp/manage-user-network.yml
   when:
   - hosting_infrastructure == 'openstack'
-  tags:
-  - 'never'
-  - 'install'
 
 - import_playbook: ../osp/provision-osp-instance.yml
   when:
   - hosting_infrastructure == 'openstack'
-  tags:
-  - 'never'
-  - 'install'
+
+- import_playbook: ../aws/provision-instance.yml
+  when:
+  - hosting_infrastructure == 'aws'
 
 - import_playbook: ../rhsm.yml
-  tags:
-  - 'never'
-  - 'install'
+  when:
+  - rhsm_register|default(False)
```

Comment:
The `rhsm` playbook already checks the `rhsm_register` variable, so this when-check shouldn't be necessary - i.e.: https://github.com/redhat-cop/infra-ansible/blob/master/playbooks/rhsm.yml#L14

Reply:
Agreed.  I removed it.